.github/workflows/ci.yml

```diff
@@ -195,3 +195,6 @@ jobs:
 
       # Check the formatting of all packages:
       - run: cd esp-config && cargo test --features build
+
+      # Check the formatting of all packages:
```

Comment:
This comment is a bit off, you're calling `cargo test` here, not checking formatting 👀 Same above, I guess.

Reply:
yeah I obviously copied all of that w/o reading the comment - can "fix" both